perf8/tests/ademo.py

```diff
@@ -82,7 +82,7 @@ async def main():
             # blocks for 2 secs
             time.sleep(2)
 
-        t3 = asyncio.get_running_loop().run_in_executor(None, some_math, i)
+        t3 = asyncio.create_task(some_math(i))
```

Comment:
This was causing `Bus error: 10`. Unclear to me why, and also unclear to me why it originally wasn't creating a task.  This seems logically equivalent.